lib/math/quat.h

```diff
@@ -72,7 +72,7 @@ class Quat {
 	Vector3 get_euler_yxz() const;
 
 	void set_euler(const Vector3& p_euler) { set_euler_yxz(p_euler); };
-	Vector3 get_euler() const { return get_euler_yxz(); };
+	Vector3 get_euler() const { return get_euler_xyz(); };
```

Comment:
Instead of modifying `get_euler()` you should rather use the `get_euler_xyz()` in your code.